AutoCollection/CorrelationContextManager.ts

```diff
@@ -116,30 +124,23 @@ export class CorrelationContextManager {
             return;
         }
 
-        // Run patches for Zone.js
         if (!CorrelationContextManager.hasEverEnabled) {
             this.hasEverEnabled = true;
 
-            // Load in Zone.js
             try {
-                // Require zone if we can't detect its presence - guarded because of issue #346
-                // Note that usually multiple requires of zone.js does not error - but we see reports of it happening
-                // in the Azure Functions environment.
-                // This indicates that the file is being included multiple times in the same global scope,
-                // averting require's cache somehow.
-                if (typeof Zone === "undefined") {
-                    require("zone.js");
+                if (typeof this.cls === "undefined") {
+                    this.cls = require('cls-hooked');
                 }
             } catch (e) {
-                // Zone was already loaded even though we couldn't find its global variable
-                Logging.warn("Failed to require zone.js");
+                // cls-hooked was already loaded even though we couldn't find its global variable
```

Comment:
It looks like cls-hooked doesn't use a global ambient variable like Zone did. This try/catch might not be necessary. Even if we keep it, the comment is now slightly wrong :smile: